src/elements/content-sidebar/activity-feed/annotations/AnnotationActivity.js

```diff
@@ -0,0 +1,186 @@
+// @flow
+import classNames from 'classnames';
+import noop from 'lodash/noop';
+import React, { useState } from 'react';
```

Comment:
Can we use `import * as React from 'react';`? I think it can allow easier mocking of hooks.